docker-compose.yml

```
@@ -0,0 +1,23 @@
+      - .:/app
+      - /app/node_modules
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:3000/health"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 40s
+
+volumes:
+  redis_data: 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)